api/src/RaceResults.sln3

```diff
@@ -0,0 +1,88 @@
+﻿
```

Comment:
what is this file? what is a .sln3 extension? why is this here?

Reply:
Good catch. Forgot to delete